### DISCUSSION TITLE
Constant error during the Communicate using Markdown learning path

### DISCUSSION THREAD

**Discussion by AshutoshDhungana** [community/community#101893] [GitHub Education]
### Hi everyone,

##During the learning path "Communicate using Markdown", on the step 3: Add a code example, it says to use ``` to enclose the lines of code and thats what i have done but i get constant error emails (picture down below. i have tried every possible way of editing the code to make it work but its the same thing. 

![Screenshot (3205)](https://github.com/community/community/assets/63156807/2d2bfef9-0b8c-4d72-b977-af2b363db0e4)
![Screenshot (3207)](https://github.com/community/community/assets/63156807/732a55fd-cb43-4d8a-8671-61ec14170dad)
![Screenshot (3206)](https://github.com/community/community/assets/63156807/3224356c-a402-4de8-ae2b-7aa38ee6b684)

**Comment by Flavio13Lins** [comment]
Try to remove the 'x' inside the boxes.
